src/main/kotlin/app/revanced/util/fingerprint/FingerprintUtils.kt

```diff
@@ -0,0 +1,21 @@
+package app.revanced.util.fingerprint
+
+import app.revanced.patcher.fingerprint.method.impl.CustomFingerprint
+import org.jf.dexlib2.Opcode
+import org.jf.dexlib2.iface.instruction.Instruction
+
+object FingerprintUtils {
+    inline fun <reified T : Instruction> hasInstruction(
```

Comment:
Following the naming convention:

```suggestion
    inline fun <reified T : Instruction> any(
```